style.css

```diff
@@ -0,0 +1,1651 @@
+  line-height: 24px;
+}
+
+.aday4_P {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.bomult {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: absolute;
+  width: 295px;
+  height: 72px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  align-items: center;
+}
+
+.multhtml {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 57px;
+  height: 24px;
+  border-radius: 8px;
+  background: #ebebff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  left: 16px;
+  right: 254px;
+  top: 428;
+  bottom: 100px;
+  list-style: none;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  justify-content: center;
+}
+
+.multcss {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 45px;
+  height: 24px;
+  border-radius: 8px;
+  background: #ebebff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  left: 675px;
+  right: 436px;
+  top: 244;
+  bottom: 228px;
+  list-style: none;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  justify-content: center;
+}
+
+.multJava {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 89px;
+  height: 24px;
+  border-radius: 8px;
+  background: #ebebff;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  left: 728px;
+  right: 339px;
+  top: 244;
+  bottom: 228px;
+  list-style: none;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  justify-content: center;
+}
+
+.buttonmult {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: absolute;
+  width: 295px;
+  height: 72px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  align-items: center;
+}
+
+.go-buttonmult {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  gap: 10px;
+  box-sizing: border-box;
+  width: 126px;
+  height: 48px;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  letter-spacing: 0.03em;
+  border: solid #396df2;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  border-radius: 8px;
+  background-color: #fff;
+  text-align: center;
+  line-height: 24px;
+  color: #396df2;
+  transition: transform 0.4s ease-in-out;
+}
+
+.go-buttonmult:hover {
+  transform: scale(1.1);
+  background-color: rgb(57, 109, 242);
+  color: rgb(223, 225, 230);
+}
+
+.contact-me {
+  display: flex;
+  justify-content: start;
+  box-sizing: border-box;
+  width: 375px;
+  height: 1263px;
+  left: 375px;
+  top: 4769px;
+  background-color: #fff;
+  border-top-right-radius: 80px;
```

Comment:
- [ ] It seems there is something else behind the About section, please remove it to be able to see the upright corner (I changed the color to prove it)

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/1b876b1a-4265-439f-a3f5-f9cdd8ba2791)

Reply:
ok. Perfect.

The color was made under the requirements described before.

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/f72a8498-bdb6-480a-96bb-91aeb629dfb0)